app-analyses/src/main/kotlin/com/usvision/analyses/detector/DependentHubInstance.kt

```diff
@@ -0,0 +1,10 @@
+package com.usvision.analyses.detector
+
+import com.usvision.model.domain.Microservice
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class DependentHubInstance(
+    val microservice: Microservice,
+    val numberOfDependencies: Int
```

Comment:
[Para ponderar, não é um pedido]
Vale a pena colocar a lista de todas as dependências ao invés do número simples?

Reply:
Faz sentido sim.
Criei um item para aplicar esse item depois das implementações que estamos realizando.

https://github.com/the-sortinghat/pattern-detector/issues/9